### DISCUSSION TITLE
Notification not showing content that was loaded

### DISCUSSION THREAD

**Discussion by lawndoc** [Windos/BurntToast#169] [Q&A]
Hi, first of all I want to say how awesome this module is. I've been playing around with it for the past day and it's great.

In my initial testing, I would sometimes run into an issue where the custom logo that I selected wouldn't render and the notification's image would be blank. It would happen infrequently and usually trying again would fix it.

Now that I'm becoming a power toaster, I've run into a scenario where that missing content issue is even worse. Below is my script (modules are included locally with my script):

```powershell
# set title, message, and url
param (
    $title = "Test Security Alert",
    $message = "This is a test message that would normally contain content about a phishing campaign or another urgent security alert.",
    $url = "https://www.google.com",
    $from = "IT Security"
)

# load BurntToast
Import-Module -Name $PSScriptRoot\BurntToast\BurntToast.psm1 -Force
Import-Module -Name $PSScriptRoot\RunAsUser\runasuser.psm1 -Force

# load script to run as logged in user
$notification = {
    # create text objects for the message content
    $text1 = New-BTText -Content $title
    $text2 = New-BTText -Content $message

    # set image for notification logo
    $image1 = New-BTImage -Source "$PSScriptRoot\BurntToast\Images\MyLogo.png" -AppLogoOverride -Crop Circle

    # select the audio tone that should be played when the notification displays
    # suppress warning about -Path becoming obsolete in BurntToast v0.9.0
    $prefBackup = $WarningPreference
    $WarningPreference = 'SilentlyContinue'
    $audio1 = New-BTAudio -Path 'C:\Windows\media\Windows Foreground.wav'
    $WarningPreference = $prefBackup

    # assemble the notification object
    $attrib = [Microsoft.Toolkit.Uwp.Notifications.ToastGenericAttributionText]::new()
    $attrib.Text = "from $from"
    $binding1 = New-BTBinding -Children $text1, $text2 -AppLogoOverride $image1 -Attribution $attrib
    $visual1 = New-BTVisual -BindingGeneric $binding1
    $content1 = New-BTContent -Visual $visual1 -Audio $audio1 -Scenario IncomingCall -Launch $url -ActivationType Protocol

    # submit the notification object to be displayed
    Submit-BTNotification -Content $content1 -UniqueIdentifier "$from Notification"
}

invoke-ascurrentuser -scriptblock $notify -NoWait
```

When I execute the script as System, the notification pops up, but it looks like this:

![image](https://user-images.githubusercontent.com/24688343/117329445-42722a80-ae5a-11eb-81ac-13d0cd020bda.png)

Does anyone else ever see this or have any ideas what's going on here?

**Comment by lawndoc** [comment]
One thing I forgot to clarify: the part that I crossed out was what I had set as the AppID in the local config.json

**Comment by Windos** [comment]
I'm wondering if SYSTEM has a different opinion on `$PSScriptRoot` path than your user account, so the image file doesn't exist at `$PSScriptRoot\BurntToast\Images\MyLogo.png`

Also remember that depending on how you're running the code, `$PSScriptRoot` can be `$null`

One other note, though it shouldn't impact this.

`New-BTAudio -Path 'C:\Windows\media\Windows Foreground.wav'` isn't working as you probably think it is unless you're using an ancient version of Windows 10. They deprecated the use of custom sounds (even if that's the Windows WAV files).

Try `New-BTAudio -Source` instead.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
